server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -531,11 +540,16 @@ public SnapshotsInProgress(StreamInput in) throws IOException {
                 // generations.
                 version = in.readBoolean() ? SnapshotsService.SHARD_GEN_IN_REPO_DATA_VERSION : SnapshotsService.OLD_SNAPSHOT_FORMAT;
             }
+            List<String> dataStreams = Collections.emptyList();
```

Comment:
NIT: Maybe refactor this a little cleaner to:

```java
final  List<String> dataStreams;
if (in.getVersion().onOrAfter(DATA_STREAMS_IN_SNAPSHOT)){
                dataStreams = Arrays.asList(in.readStringArray());
} else {
  dataStreams = Collections.emptyList();
}
```

Reply:
Done